homeassistant/components/backup/agent.py

```diff
@@ -27,7 +27,7 @@ class BackupAgentUnreachableError(BackupAgentError):
     _message = "The backup agent is unreachable."
 
 
-class BackupNotFound(BackupAgentError):
+class BackupNotFound(BackupAgentError, BackupManagerError):
```

Comment:
I think we should move this definition to `models.py` since it's not only a `BackupAgentError` anymore but also a `BackupManagerError`, and then also move `BackupAgentError` there to make it work. Alternatively move all exceptions to a new module `exceptions.py`.